src/ClientOperations/AbstractSubscriptionOperation.php

```diff
@@ -91,7 +91,11 @@ public function __construct(
 
     protected function enqueueSend(TcpPackage $package): void
     {
-        ($this->getConnection)()->enqueueSend($package);
+        $connection = ($this->getConnection)();
+
+        if (null !== $connection) {
+            $connection->enqueueSend($package);
```

Comment:
Is it alright to just completely ignore the package here? Why?

Reply:
Usually this only happens when you want to ack a message and the connection just went away. On appending to a stream with a closed connection, this would not happen, because your promise would not be resolved.